vnext/Microsoft.ReactNative/Modules/AlertModule.cpp

```diff
@@ -61,7 +61,17 @@ void Alert::ProcessPendingAlertRequests() noexcept {
 
   if (Is19H1OrHigher()) {
     // XamlRoot added in 19H1
-    if (const auto xamlRoot = React::XamlUIService::GetXamlRoot(m_context.Properties().Handle())) {
+    const auto xamlUIService = React::XamlUIService::FromContext(m_context.Handle());
+    xaml::XamlRoot xamlRoot = nullptr;
+    if (args.rootTag) {
```

Comment:
note we've been using InvalidTag = -1 
https://github.com/microsoft/react-native-windows/blob/main/vnext/Microsoft.ReactNative/XamlView.h#L12